imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/ExoPlayerWithAdPlayback.java

```diff
@@ -506,7 +508,7 @@ public void setContentProgressProvider(final com.kaltura.playkit.Player contentP
             long position = contentPlayer.getCurrentPosition();
             //log.d("getContentProgress getDuration " +  duration);
             //log.d("getContentProgress getCurrentPosition " + position);
-            if (position > 0 && duration > 0 && position >= duration && adCuePoints != null && !adCuePoints.hasPostRoll()) {
+            if (duration > 0 && position >= duration && adCuePoints != null && !adCuePoints.hasPostRoll()) {
```

Comment:
why this logic has to be changed?

Reply:
Not a must it was, actually position > 0 was not mandatory because if duration is >0 then we are checking it position >= duration which will anyways be 0 after first check.